### PR TITLE
add detector name branch to all scans

### LINKED ISSUE #273: Feature Request: Add TBranch to output TTree created by sbitRateScanAllLinks(...) to store Detector Serial Number
## Brief summary of issue

Right now the data taken by function [`sbitRateScanAllLinks()`](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/rpc-playground/utils/scanUtils.py#L378-L566) contains multiple OH's but does not include the detector serial number in the output path.  This makes the risk that some later time this information could be lost.  Request to add a TBranch in the output file that stores this information.

### Types of issue

- [ ] Bug report (report an issue with the code)
- [x] Feature request (request for change which adds functionality)

## Expected Behavior

The `sbitRateScanAllLinks` would need to import `chamber_config` dictionary or take it as an input argument (as [`sbitRateAnalysis`](https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/9ce5139a7d9b1a657646e00b8753ea52807286c4/utils/threshAlgos.py#L1139-L1453) does).  This would provide the mapping of geographic address to chamber name.

Then the [`gemSbitRateTreeStructure`](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/15cf95544f2c2e0d98864ae8a65f166d80f9d245/utils/treeStructure.py#L213-L266) object would need a data member for storing chamber name and an associated `TBranch`.

This would then be filled at in `sbitRateAnalysis` at:

 - VFAT level: [L496-L506](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/15cf95544f2c2e0d98864ae8a65f166d80f9d245/utils/scanUtils.py#L496-L506)
 - Overall rate: [L525-L535](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/15cf95544f2c2e0d98864ae8a65f166d80f9d245/utils/scanUtils.py#L525-L535)

## Current Behavior

Detector serial number is not stored in output raw file.

## Context (for feature requests)

See above.

## Your Environment

* Version used: 15cf95544f2c2e0d98864ae8a65f166d80f9d245
* Shell used:

### PR DESCRIPTION
**Pull request by AndrewLevin**
Adds a detector name (like `GE11-X-S-INDIA-0002`) branch to all scans.

Closes #273 

## Description
Creates a new branch that is a vector of strings and pushes back a single dummy string "X-X-X-X-X". In `setDefaults` function, import the `chamber_config` and set the detector name. Update s`dacScanAllLinks` and `sbitRateScanAllLinks` such that it sets the OH-specific detector name when it fills the output tree.

### Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Motivation and Context
Intended to resolve issue https://github.com/cms-gem-daq-project/vfatqc-python-scripts/issues/273

## How Has This Been Tested?
Yes, this was tested on the coffin with several scan types: http://cmsonline.cern.ch/cms-elog/1090191

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: utils/treeStructure.py
Comment (jsturdy) [inline utils/treeStructure.py:108]: Here is likely where we could consider thinking of how to pull the name out of the DB

**Review by jsturdy — DISMISSED** [review]
Looks OK, though all of the duplication smells like a candidate for consolidation

**Comment by AndrewLevin** [comment]
I just realized that there are additional changes needed in `scanUtils`. Please do not merge right now.

File: utils/treeStructure.py
Comment (AndrewLevin) [reply utils/treeStructure.py:108]: I get the error

```
cx_Oracle.DatabaseError: ORA-01017: invalid username/password; logon denied
```

when I try to access the DB in my virtual environment. Is there a trick to access the DB in a virtual environment?

**Comment by jsturdy** [comment]
> I just realized that there are additional changes needed in `scanUtils`. Please do not merge right now.

OK, `rebase` and remove `WIP` when you're ready for review

**Comment by AndrewLevin** [comment]
Done.

**Review by jsturdy — APPROVED** [review]

**Pull request merged**
